swap_meet/vendor.py

```diff
@@ -1,2 +1,104 @@
+
+####################### WAVE 1 #############################
+
 class Vendor:
-    pass
\ No newline at end of file
+    # inventory is an empty list
+    def __init__(self, inventory=None):
+
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        if item:
+            self.inventory.append(item)
+            return item
+        
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
+
+
+####################### WAVE 2 #############################
+    def get_by_id(self, id): 
+
+        for item in self.inventory:
+            if id == item.id:
+                return item
+        
+        return None
```

Comment:
A function will automatically return `None` if there's no final return statement but I personally like an explicit `return None` if that is indeed the case! Other people may have different preferences, and that's fine too! 

I think what is more helpful is that your code preferences are consistent thought your files -- it would be nice to also explicitly return `None` for the `add()` method on L12 if this is indeed your preference!